core/src/main/java/com/netflix/iceberg/hadoop/HadoopTables.java

```diff
@@ -66,6 +70,28 @@ public Table load(String location) {
     return new BaseTable(ops, location);
   }
 
+  @Override
+  public void drop(String location) {
```

Comment:
This doesn't seem to do any checking if this location is actually a valid Iceberg table.